legal-api/report-templates/annualReport.html

```diff
@@ -19,7 +19,7 @@
     <div class="column-report">
       <div class="column-report-headspace"></div>
       <div class="header-blue">
-        {{ annualReport.annualReportDate[:4] }} BC COOPERATIVE ANNUAL REPORT
+        {{ header.ARFilingYear }} BC COOPERATIVE ANNUAL REPORT
```

Comment:
does `ARFilingYear` is been set to header from UI

Reply:
I believe all header props are available. I tested (see below) to make sure :)